tests/formulagrader/test_matrixgrader.py

```diff
@@ -1,7 +1,7 @@
 from __future__ import print_function, division, absolute_import
 
-from pytest import raises
 import re
+from pytest import raises
```

Comment:
Really? That order mattered???

Reply:
No, it didn't matter. But pylint likes it when standard-library imports come before other imports, and it was easy to change.

(It's the lowest category of pylint message.)